DataConnectors/UploadToBlob-Whitelist/readme.md

```diff
@@ -0,0 +1,47 @@
+# Deploy Function App for Uploading Files to Azure Blob Storage
+This function app will downwload files from Urls and upload to Blog storage for centralized whitelisting.
+
+## Deployment and Configuration
+
+```
+
+### Deploy the Function App
+There are 2 deployment Options.
+
+#### 1: Deploy via Azure ARM Template
+1.  Deploy the template.
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FDataConnectors%2FUploadToBlob-Whitelist%2Fazuredeploy.json" target="_blank">
+    <img src="https://aka.ms/deploytoazurebutton""/>
+</a>
+
+#### 2: Deploy via VS Code
+Note: You will need to prepare VS code for Azure function development.  See https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-function-powershell#prerequisites
```

Comment:
URLs for documentation need to have locale removed - https://docs.microsoft.com/azure/azure-functions/functions-create-first-function-powershell#prerequisites